CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Change log for kotlinx.coroutines
 
+## Version 1.3.0
+
+### Flow
+
+This version is the first stable release with [`Flow`](https://kotlin.github.io/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/-flow/index.html) API.
+
+All `Flow` API not marked with `@FlowPreview` or `@ExperimentalCoroutinesApi` annotations is stable and here to stay.
+Flow declarations marked with `@ExperimentalCoroutinesApi` has [the same guarantees](/docs/compatibility.md#experimental-api) as regular experimental API.
+Please note that API marked with `@FlowPreview` has [weak guarantees](/docs/compatibility.md#flow-preview-api) on source, binary and semantic compatibility.
```

Comment:
"API has" reads funny for me. I'd replace it is with "APIs have"

Reply:
I was using stdlib style. E.g. "API is" in https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/src/kotlin/time/ExperimentalTime.kt#L13

Reply:
It is a different context. When you are looking at a specific API has see it annotated as @SomethingExpreimental, then you go do checkout the docs for @SomethingExperimental and see "this API is ...". It makes sense. However, in release statement you are making a more general statement about all annotated APIs.